### ISSUE TITLE
CLI should validate that project has been "init"ed

### ISSUE THREAD

**Issue by raubitsj** [wandb/client#291]
* Weights and Biases version: 0.6.34
* Python version: 3.6.4
* Operating System: Linux

### Description

Attempting to run wandb sweep without initializing the project.
Also specifying dryrun which probably shouldn't work with sweeps.

### What I Did

```
$ WANDB_MODE=dryrun wandb sweep sweep.yaml 
Creating sweep from: sweep.yaml
wandb: Network error (HTTPError), entering retry loop. See wandb/debug.log for full traceback.
```
The failure is because entity was not set.

**Comment by raubitsj** [comment]
Related to this, if you attempt to create a sweep when the project does not exist, you get a less than friendly error:
```
Creating sweep from: sweep.yaml
wandb: Network error (HTTPError), entering retry loop. See wandb/debug.log for full traceback.

head wandb/debug.log 
2019-01-24 14:41:59,443 DEBUG   MainThread:21791 [connectionpool.py:_new_conn():813] Starting new HTTPS connection (1): api.wandb.ai:443
2019-01-24 14:41:59,586 DEBUG   MainThread:21791 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 404 133
```

**Comment by raubitsj** [comment]
Fixed in 0.7.1
